webpack.config.js

```diff
@@ -13,48 +13,25 @@ let rules = [
   }
 ];
 
-let main = {
-  entry: `./source/js/main.js`,
-  output: {
-    path: frontendPath,
-    filename: `main.compiled.js`
-  },
-  module: {
-    rules
-  }
-};
-
-let paymentsCard = {
-  entry: `./source/js/payments-card.js`,
-  output: {
-    path: frontendPath,
-    filename: `payments-card.compiled.js`
-  },
-  module: {
-    rules
-  }
-};
+function generate(name) {
+  return {
+    entry: `./source/js/${name}.js`,
+    output: {
+      path: frontendPath,
+      filename: `${name}.compiled.js`
+    },
+    module: {
+      rules
+    },
+    devtool: `none` // see https://webpack.js.org/configuration/devtool/
```

Comment:
The `devtool` instruction was missing, which means it defaulted to creating a source map based on `eval`, which made _everything_ break.

Fuck magical implicit webpack nonsense, it absolutely destroys a day's worth of productivity.